src/mesh/caching-mesh-adapter-decorator.ts

```diff
@@ -49,6 +50,14 @@ export class CachingMeshAdapterDecorator implements MeshAdapter {
     }
   }
 
+  async updateMeshTenant(
+    updatedTenant: MeshTenant,
+    originalTenant: MeshTenant,
+  ): Promise<void> {
+    await this.meshAdapter.updateMeshTenant(updatedTenant, originalTenant);
```

Comment:
t: hopefully every failed attempt to update a mesh tenant results in an exception so the update of the cache never happens when something fails.

Reply:
I rely on the `checkForErrors` method in our CLI facades for that ^^

I tested with invalid / too long tags and that all works out.